pages.it/linux/ip.md

```diff
@@ -0,0 +1,36 @@
+# ip
+
+> Mostra / manipola il routing, i dispositivi, i criteri di routing e i tunnel.
+> Alcuni sottocomandi, come `ip address`, hanno una propria documentazione d'uso. Maggiori informazioni: <https://www.man7.org/linux/man-pages/man8/ip.8.html>
+
+- Lista delle interfacce con informazioni dettagliate:
+
+`ip address`
+
+- Lista delle interfacce con informazioni brevi sul livello di rete:
+
+`ip -brief address`
+
+- Lista delle interfacce con informazioni brevi sul livello di collegamento:
+
+`ip -brief link`
+
+- Visualizza la tabella di routing:
+
+`ip route`
+
+- Mostra i vicini (tabella ARP):
+
+`ip neighbour`
+
+- Impostare un'interfaccia up/down:
+
+`ip link set {{interfaccia}} up/down`
+
+- Aggiungere/eliminare un indirizzo ip a un'interfaccia:
+
+`ip addr add/del {{ip}}/{{mask}} dev {{interfaccia}}`
+
+- Aggiungere una rotta predefinita:
```

Comment:
```suggestion
- Aggiungi una route predefinita:
```
We can use `route` in Italian too, for clarity. It's common to use "route" and "routing" in Italian too.